.github/workflows/lint-php.yml

```
@@ -0,0 +1,65 @@
+name: Lint PHP
+
+on:
+  workflow_call:
+    inputs:
+      PHP_MATRIX:
+        description: Matrix of PHP versions as a JSON formatted object.
+        default: '["8.0"]'
+        required: false
+        type: string
+      COMPOSER_ARGS:
+        description: Set of arguments passed to Composer.
+        default: '--prefer-dist'
+        required: false
+        type: string
+      LINT_ARGS:
+        description: Set of arguments passed to PHP-Parallel-Lint.
+        default: '-e php --colors --show-deprecated .'
```

Not sure if this works (the example `LINT_ARGS` in the documentation does it), but we should move the target folder in front of the flags so that we can add the `cs2pr` stuff hardcoded to the end.
```suggestion
        default: '. -e php --colors --show-deprecated'
```